### PR TITLE
add post-download page

### PR DESCRIPTION
**Pull request by gdams**
##### Checklist

- [x] `npm test` passes
- [ ] documentation is changed or added (if applicable)
- [ ] permission has been obtained to add new logo (if applicable)
- [x] contribution guidelines followed [here](https://github.com/adoptium/adoptium.net/blob/master/CONTRIBUTING.md)

### CODE REVIEW

File: src/handlebars/download.handlebars [hunk 1]
```diff
@@ -0,0 +1,23 @@
+{{> header title='Thank You | ' }}
+
+<main class="grey-bg">
+   <h1 class="margin-bottom large-title">Thank you for downloading</h1>
+   <div id="download">
+      <script id="template-selector" type="text/x-handlebars-template">
+         \{{#if link}}
+         <meta http-equiv="refresh" content="0; url=\{{link}}">
+         <h3>You're download should begin shortly... Click <a href=\{{link}}>here</a> to start the download manually.</h3>
```
Comment (tellison) [inline src/handlebars/download.handlebars]:
```suggestion
         <h3>Your download will begin shortly... Click <a href=\{{link}}>here</a> to start the download manually.</h3>
```

File: src/handlebars/download.handlebars [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+         \{{else}}
+         <h3>Ooops - something doesn't seem quite right here. Please try again.</h3>
+         \{{/if}}
+         <h2>The Adoptium Working Group relies on donations from our generous members/sponsors.</h2>
+         <div class="home-buttons">
+            <a class="dl-button a-button home-button-left" href="./join.html"><span>Join the Working group</span></a>
+            <a class="dl-button a-button home-button-right" href="./members.html"><span>View our Members</span></a>
```
Comment (tellison) [inline src/handlebars/download.handlebars:21]: IMHO the first half should be a "donate" button (linking to a TBD location, but start with https://www.eclipse.org/donate/), and the second part be "join"

File: src/handlebars/download.handlebars (hunk 1 above)
Comment (gdams) [reply src/handlebars/download.handlebars]: I've tweaked the wording now, thanks!

File: src/handlebars/download.handlebars (hunk 2 above)
Comment (gdams) [reply src/handlebars/download.handlebars:21]: Updated

File: src/handlebars/download.handlebars [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+   <h1 class="margin-bottom large-title">Thank you for your download!</h1>
+   <div id="download" class="support">
+      <script id="template-selector" type="text/x-handlebars-template">
+         \{{#if link}}
+         <meta http-equiv="refresh" content="0; url=\{{link}}">
+         <h3 style="color: #7c7a8f">If the download doesn't start in a few seconds, please <a href=\{{link}}>click here</a> to start the download.</h3>
+         \{{else}}
+         <h3>Ooops - something doesn't seem quite right here. Please try again.</h3>
+         \{{/if}}
+         <p>Eclipse Temurin binaries are 100% free and open source and used by millions of developers every day. Here are three easy ways you can contribute toward the future development of Eclipse projects and technologies.</p>
```
Comment (karianna) [inline src/handlebars/download.handlebars:13]: Maybe: Eclipse Temurin binaries (by the Eclipse Adoptium Project)...

Maybe: development of Eclipse.... --> development of Eclipse Adoptium....

File: src/handlebars/download.handlebars [hunk 4]
```diff
@@ -0,0 +1,28 @@
...
+         <h2>Donate to the Eclipse Adoptium Working Group</h2>
+         <p>You can make a one-time of regular targetted donation to the Adoptium Working Group by filling in the <a href="https://www.eclipse.org/org/workinggroups/sponsorship/working-group-sponsorship-agreement.pdf">Working Group Sponsorship Agreement.</a></p>
```
Comment (karianna) [inline src/handlebars/download.handlebars:15]: s/of/or

File: src/handlebars/download.handlebars [hunk 5]
```diff
@@ -0,0 +1,28 @@
...
+         <h2>Get involved</h2>
+         <p>Whether you choose to report bugs, request features, or become a committer, you can help improve the technology for yourself and the rest of the community.</p>
+         <h2>Join the Working Group</h2>
+         <div class="home-buttons">
+            <a class="dl-button a-button home-button-left" href="./join.html"><span>Join the Working group</span></a>
```
Comment (karianna) [inline src/handlebars/download.handlebars:20]: Do we want the more direct join us on GitHub and Slack as well?

File: src/handlebars/download.handlebars [hunk 6]
```diff
@@ -0,0 +1,28 @@
...
+         <meta http-equiv="refresh" content="0; url=\{{link}}">
+         <h3 style="color: #7c7a8f">If the download doesn't start in a few seconds, please <a href=\{{link}}>click here</a> to start the download.</h3>
+         \{{else}}
+         <h3>Ooops - something doesn't seem quite right here. Please try again.</h3>
+         \{{/if}}
+         <p>Eclipse Temurin binaries are 100% free and open source and used by millions of developers every day. Here are three easy ways you can contribute toward the future development of Eclipse projects and technologies.</p>
+         <h2>Donate to the Eclipse Adoptium Working Group</h2>
+         <p>You can make a one-time of regular targetted donation to the Adoptium Working Group by filling in the <a href="https://www.eclipse.org/org/workinggroups/sponsorship/working-group-sponsorship-agreement.pdf">Working Group Sponsorship Agreement.</a></p>
+         <h2>Get involved</h2>
+         <p>Whether you choose to report bugs, request features, or become a committer, you can help improve the technology for yourself and the rest of the community.</p>
```
Comment (karianna) [inline src/handlebars/download.handlebars:17]: The word community is a hanging widow in the preview

File: src/scss/styles-1-large-main.scss [hunk 7]
```diff
@@ -729,3 +729,23 @@ footer#solstice-footer a {
    color: black;
 }
 
+.home-buttons a {
+  margin: 0 !important;
+  width: 12em !important;
+}
+
+.home-button-left {
```
Comment (karianna) [inline src/scss/styles-1-large-main.scss:737]: The buttons don't have enough padding given the text

**Review by karianna — CHANGES_REQUESTED** [review]

**Pull request closed without merging**